addOns/sequence/src/main/zapHomeFiles/policies/sequence.policy

```diff
@@ -0,0 +1,313 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<configuration>
+    <policy>sequence</policy>
+    <scanner>
+        <level>OFF</level>
+        <strength>MEDIUM</strength>
+    </scanner>
+    <plugins>
+        <p6>
+            <enabled>false</enabled>
+            <level>OFF</level>
```

Comment:
Strictly speaking we dont need the `OFF` ones any more, as the default level is `OFF`.

Reply:
Can we get away wihout the `<enabled>false</enabled>` as well?